docs/src/models/biological.md

```diff
@@ -1,166 +1,307 @@
+rn = @reaction_network begin
+  (2.0,1.0) X + Y ↔ XY               
 end
 ```
-
-Parameters can be added to the network by declaring them after the reaction
-network. Parameters can only exist in the reaction rate and not as a part of the
-reaction.
-
+which is identical to
 ```julia
-rn = @reaction_network rType begin
-    (kB, kD), X + Y ↔ XY
-end kB kD
-p = [2.0, 1.0]
+rn = @reaction_network begin
+  2.0, X + Y → XY             
+  1.0, X + Y ← XY          
+end
 ```
 
-The parameter set `p` must be passed to the problem constructor. The parameter
-values can be changed after the reaction network is defined.
-
-The reaction rate do not need to be constant, but maybe depend on the
-concentration of the reactants.
-
+#### Combining several reactions in one line
+Several similar reactions can be combined in one line by providing a tuple of reaction rates and/or substrates and/or products. If several tuples are provided they much all be of identical length. These pairs of reaction networks are all identical:
 ```julia
-rn = @reaction_network rType begin
-    (1.0,2XY), X + Y ↔ XY
+rn1 = @reaction_network begin
+  1.0, S → (P1,P2)               
+end
+rn2 = @reaction_network begin
+  1.0, S → P1     
+  1.0, S → P2 
 end
 ```
-
-The hill function `hill(x,v,K,n) = v*(x^n)/(x^n + K^n)` can be used, as well as
-the Michaelis Menten function (the hill function with `n = 1`).
-
 ```julia
-rn = @reaction_network rType begin
-    (1.0,hill(XY,1.5,2.0,2)), X + Y ↔ XY
+rn1 = @reaction_network begin
+  (1.0,2.0), (S1,S2) → P             
+end
+rn2 = @reaction_network begin
+  1.0, S1 → P     
+  2.0, S2 → P
+end
+```
+```julia
+rn1 = @reaction_network begin
+  (1.0,2.0,3.0), (S1,S2,S3) → (P1,P2,P3)        
+end
+rn2 = @reaction_network begin
+  1.0, S1 → P1
+  2.0, S2 → P2   
+  3.0, S3 → P3  
+end
+```
+This can also be combined with bi-directional arrows in which case separate tuples can be provided for the backward and forward reaction rates separately. These reaction networks are identical
+```julia
+rn1 = @reaction_network begin
+ (1.0,(1.0,2.0)), S ↔ (P1,P2)  
+end
+rn2 = @reaction_network begin
+  1.0, S → P1
+  1.0, S → P2
+  1.0, P1 → S   
+  2.0, P2 → S 
 end
 ```
 
-By using the `@reaction_func` macro it is possible to define your own functions,
-which may then be used when creating new reaction networks.
+#### Production and Destruction and Stoichiometry
+Sometimes reactants are produced/destroyed from/to nothing. This can be designated using either `0` or `∅`:
+```julia
+rn = @reaction_network begin
+  2.0, 0 → X
+  1.0, X → ∅
+end
+```
+Sometimes several molecules of the same reactant is involved in a reaction, the stoichiometry of a reactant in a reaction can be set using a number. Here two species of `X` forms the dimer `X2`:
+```julia
+rn = @reaction_network begin
+  1.0, 2X → X2
+end
+```
+this corresponds to the differential equation:
+d[X]/dt = -1.0\*[X\]^2/2!
+d[X2]/dt = 1.0\*[X\]^2/2!
```

Comment:
```math
\begin{align*}
\frac{dX}{dt} =& -2 \cdot \frac{X^{2}}{2} \\
\frac{dX2}{dt} =& \frac{X^{2}}{2} \\
\end{align*}
```
surrounded by ```math

Reply:
This PR is a bit outdated, but you seem to be right that some of these are missing.

How do you meant though, I try to check how
´´´
\begin{align*}
\frac{\mathrm{d}\left[X\right]}{dt} =&  - 2.0 \cdot \left[ X \right] \cdot \left[ Y \right] \\
\frac{\mathrm{d}\left[Y\right]}{dt} =&  - 2.0 \cdot \left[ X \right] \cdot \left[ Y \right] \\
\frac{\mathrm{d}\left[XY\right]}{dt} =& 2.0 \cdot \left[ X \right] \cdot \left[ Y \right] - \left[ XY \right] \\
\frac{\mathrm{d}\left[Z1\right]}{dt} =& XY \\
\frac{\mathrm{d}\left[Z2\right]}{dt} =& XY \\
\end{align*}
´´´
would look like (but with ´´´math before and simple three ´ after), but it does not look very LaTeXy?

(I try to preview it here and paste it into a discourse message and see how it looks, the only way I know how to actually look how .md formatting turns out like).

Looking at the big one it looks perfectly fine at
http://docs.juliadiffeq.org/latest/models/biological.html
(ignoring that the new lines does not work).
but on 
https://github.com/JuliaDiffEq/DiffEqDocs.jl/blob/master/docs/src/models/biological.md
it for some reason does not work at all. Checking on my discourse message it looks perfect. I have no idea why it works somewhere and somewhere not.